### DISCUSSION TITLE
Patch for libusb until 1.0.25 is released?

### DISCUSSION THREAD

**Discussion by Andrew-J-Larson** [Homebrew/discussions#1593] [Tap maintenance and Homebrew development]
It seems as though that the current version of libusb doesn't work on macOS. But, a patch in https://github.com/libusb/libusb/pull/911 appears to have fixed the issues. I know that I've seen in other formulas where patches have been applied to packages, and wondering if the same should and could be done for libusb (and sequentially qemu too).

**Comment by carlocab** [comment]
That PR is a bit big for a patch, and it seems they might be ready to release 1.0.25 soon. Perhaps we could just follow up on that?

**Reply by Andrew-J-Larson** [reply]
I was thinking that was going to be the case, but I also noticed that [their roadmap](https://github.com/libusb/libusb/milestone/9) is due on June 30th, which is still a bit away.

**Reply by carlocab** [reply]
Does the patch generated from that PR apply to the release tarball used by the `libusb` formula? If you can extract a minimal patch required to fix `libusb`, we can maybe apply that to the existing formula.

**Reply by Andrew-J-Larson** [reply]
I mean, from what I can tell, the formula does use the latest release from the github, but that is still the 1.0.24 release. Although I did notice this: https://github.com/libusb/libusb/issues/777

Looks like they are one issue away from doing a release candidate.

**Reply by carlocab** [reply]
Yup, but we don't update formulae to release candidates, so that doesn't help very much. We'll still wait for the stable release of 1.0.25.

**Reply by Andrew-J-Larson** [reply]
In that case, is there an option in brew to update to prereleases (if not, I do know how to modify the formulae locally to do so), for when they release 1.0.25-rc1? I have the need for the USB passthrough, and want to get it as soon as I can.

**Reply by carlocab** [reply]
Short of my suggestion above:

>If you can extract a minimal patch required to fix `libusb`, we can maybe apply that to the existing formula.

Not really.
